### PR TITLE
Check ini file before loading default testnet values and add testnetdb flag

### PR DESCRIPTION
**Pull request by GalRogozinski**
# Description

In order to run private testnets it is useful to be able to control the db folder.
The `--testnet` flag has overriden user configs from ini file and disabled this option.

So a flag for a custom folder was added AND the overriding behavior of the ini is now gone.

# Fixes
https://iotaledger.atlassian.net/browse/IRI-25

## Type of change
- Bug fix (a non-breaking change which fixes an issue)

# How Has This Been Tested?

IRI was ran with flag and modified ini file.

# Checklist:

_Please delete items that are not relevant._

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by alon-e** [comment]
@galrogo is this PR still relevant if you are reworking the configuration? (#724 )
if not close it, but if it is then I agree w/ @codacy-bot 's complexity comment.

**Comment by GalRogozinski** [comment]
@alon-e 
1. It is still relevant because it will help @geminoz run testnets the second you hit the merge button.
2. This can be merged and later my work on refactoring `Configuration` will change the logic here.
3. I reduced the complexity of the code by wrapping some logic in `setTestnetConfigs()`. Because this change didn't reduce the complexity enough, but did introduce a change for @codacy-bot to pick up, a warning was created. Since the complexity of the code will be reduced by the refactoring issue, I'd rather not waste time on this matter now.

In short, can you approve and merge?
